installer/charts/rhtap-dh/templates/app-config-content.yaml

```diff
@@ -93,8 +93,21 @@ backend:
   cors:
     origin: ${BACKEND_URL}
 catalog:
-  {{- if and .Values.developerHub.RBAC.enabled $githubSecretObj }}
+  {{- if and .Values.developerHub.RBAC.enabled (or $githubSecretObj $gitlabSecretObj) }}
   providers:
+  {{- end }}
+  {{- if $gitlabSecretObj }}
```

Comment:
Keep this in alphabetical order please (ie put this section after the github section).